theme/themes/pastanaga/extras/blocks.less

```diff
@@ -622,6 +622,10 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
     .icon {
       margin-bottom: 10px !important;
     }
+
+    &.table {
```

Comment:
@sneridagh committed by accident?

Reply:
all fine, it's a small CSS fix that it was not worth the hassle to create a single PR for it... it's kind of related with the blockchooser

Reply:
I'm in get V4 final asap mode ;)